app/src/main/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModel.java

```diff
@@ -0,0 +1,111 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.ui.viewholders.ProjectSearchResultViewHolder;
+
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+import rx.subjects.PublishSubject;
+
+public interface ProjectSearchResultHolderViewModel {
+
+  final class Data {
+    final Project project;
+    final boolean isFeatured;
+
+    public Data(final @NonNull Project project, final boolean isFeatured) {
+      this.project = project;
+      this.isFeatured = isFeatured;
+    }
+  }
+
+  interface Inputs {
+    /** Call to configure the view model with a message. */
+    void configureWith(Data data);
+
+    // FIXME: add documentation
+    void onClick();
+  }
+
+  interface Outputs {
+    /** Emits key image of project */
+    Observable<String> projectImage();
+
+    /** Emits title of project */
+    Observable<String> projectName();
+
+    /** Emits a completed / days to go pair */
+    Observable<Pair<Integer, Integer>> projectStats();
+
+    // FIXME: add documentation and tests
+    Observable<Project> notifyDelegateOfResultClick();
+  }
+
+  final class ViewModel extends ActivityViewModel<ProjectSearchResultViewHolder> implements
+    ProjectSearchResultHolderViewModel.Inputs,
+    ProjectSearchResultHolderViewModel.Outputs {
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      this.configData
+        // FIXME: check for nullability on `photo`
+        .map(data -> data.isFeatured ? data.project.photo().full() : data.project.photo().med())
+        .subscribe(this.projectImage);
```

Comment:
since these are still `BehaviorSubject` outputs, which is fine for this PR, we do need to `.compose(bindToLifecycle())` before subscribing for each output